app/javascript/packs/base.jsx

```diff
@@ -52,7 +52,7 @@ window.Forem = {
     );
   },
   showModal: showWindowModal,
-  closeModal: closeWindowModal,
+  closeModal: () => closeWindowModal(),
```

Comment:
This change (suggested by @aitchiss) was to fix some unrelated broken specs that use the `<Modal />` component.